core-text/src/font.rs

```diff
@@ -206,7 +206,7 @@ pub fn new_ui_font_for_language(
     unsafe {
         let font_ref = CTFontCreateUIFontForLanguage(
             ui_type,
-            size,
+            size as CGFloat,
```

Comment:
This is needed if you build for a 32 bit target, but are there any left (that support CoreText)?

Reply:
Rust does still support some 32bit targets like `i686-apple-darwin`, `armv7s-apple-ios` and `i386-apple-ios`. And those do support CoreText.